crates/python/pyproject.toml

```diff
@@ -20,45 +20,33 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
   "Operating System :: OS Independent",
 ]
-dependencies = ["quil>=0.11.2"]
-
-# PEP 621 specifies the [project] table as the source for project metadata. However, Poetry only supports [tool.poetry]
-# We can remove this table once this issue is resolved: https://github.com/python-poetry/poetry/issues/3332
```

Comment:
Are you sure we can get rid of this? Treeline is stuck on an older version of Poetry for... reasons 😞

Reply:
Per offline discussion: this should be fine